src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java

```diff
@@ -27,15 +46,15 @@ public static void applyMappings(
      * <p>More: https://tools.hmcts.net/confluence/display/RCCD/Address+global+complex+type
      * */
     private static ImmutableMap<String, String> getAddressMapping(String prefix) {
-        String nestedFieldPrefix = StringUtils.capitalize(prefix + "Address");
+        String nestedFieldPrefix = StringUtils.capitalize(prefix + OcrSuffixes.ADDRESS);
         Map<String, String> address = new HashMap<>();
 
-        address.put(nestedFieldPrefix + "Line1", "AddressLine1");
-        address.put(nestedFieldPrefix + "Line2", "AddressLine2");
-        address.put(nestedFieldPrefix + "County", "County");
-        address.put(nestedFieldPrefix + "Postcode", "PostCode");
-        address.put(nestedFieldPrefix + "Town", "PostTown");
-        address.put(nestedFieldPrefix + "Country", "Country");
+        address.put(nestedFieldPrefix + OcrSuffixes.LINE_1, Fields.LINE_1);
+        address.put(nestedFieldPrefix + OcrSuffixes.LINE_2, Fields.LINE_2);
+        address.put(nestedFieldPrefix + OcrSuffixes.POSTCODE, Fields.POSTCODE);
+        address.put(nestedFieldPrefix + OcrSuffixes.TOWN, Fields.TOWN);
+        address.put(nestedFieldPrefix + OcrSuffixes.COUNTY, Fields.COUNTY);
+        address.put(nestedFieldPrefix + OcrSuffixes.COUNTRY, Fields.COUNTRY);
```

Comment:
`OcrSuffix` would read better than the plural form?

Reply:
I agree